sky/templates/kubernetes-ray.yml.j2

```diff
@@ -384,6 +384,14 @@ available_node_types:
               }
               trap : TERM INT; log_tail || sleep infinity & wait
 
+          {% if k8s_env_vars is not none %}
+          env:
+          {% for key, value in k8s_env_vars.items() %}
+          - name: {{ key }}
+            value: {{ value }}
+          {% endfor %}
+          {% endif %}
```

Comment:
If a user specify env in the spec, will this get overwritten?

Reply:
No, since it's a dict it will get merged